comet/archiver.py

```diff
@@ -86,17 +92,24 @@ def run(self):
             # lists
             which_list = random.choice(TYPES)
             # Do not remove from the list until it has been added to the database
-            data = self.redis.lrange(which_list, 0, 0)
+            # Use the right-most item in list since comet uses `lpush`
+            data = self.redis.lindex(which_list, -1)
 
             if not data:
-                # Wait for a moment
-                time.sleep(self.failure_wait_time)
+                # Wait for a couple of seconds since the list is empty
+                time.sleep(2)
```

Comment:
Why change this back to a magic number?

Reply:
My reasoning was just to slow down the polling rate when a list is empty since we've switched from a blocking to non-blocking check. I don't think it matters too much either way

Reply:
IThe proper way to resolve this would be to add a new attribute to `Archiver` and then move the magic number to https://github.com/chime-experiment/comet/blob/master/scripts/comet_archiver where all the other magic numbers live.